src/workerd/io/actor-cache.h

```diff
@@ -384,15 +393,8 @@ class ActorCache final: public ActorCacheInterface {
 
   private:
     // The value associated with this key. If our `valueStatus` below is `ABSENT` or `UNKNOWN`, it
-    // will have size 0.
-    //
-    // `value` cannot change after the `Entry` is constructed. When a key is overwritten, the
```

Comment:
The old comment here still provided valuable context IMO. It'd be better to clarify it with some special case (e.g. "`value` cannot change after the `Entry` is constructed and moved out of the FOO status" or something like that, which again points at the value of a different `EntrySyncStatus` for this sort of `Entry`; or of using `EntryValueStatus::UNKNOWN`, perhaps?).